Solutions/Corvus.SpecFlow.Extensions.Demo/AzureFunctionsTesting/DemoFunctionPerFeatureHooks.cs

```diff
@@ -5,9 +5,12 @@
 namespace Corvus.SpecFlow.Extensions.Demo.AzureFunctionsTesting
 {
     using System.Threading.Tasks;
-    using Corvus.SpecFlow.Extensions;
+    using Corvus.Testing.AzureFunctions;
```

Comment:
I think we need to think about naming. Although I suspect the conclusion will be that `Corvus.Testing.AzureFunctions` is a perfectly good name, and that it's `Corvus.SpecFlow.Extensions` that looks weird...

Reply:
@idg10 - see my comment above on naming - https://github.com/corvus-dotnet/Corvus.SpecFlow.Extensions/pull/83#issuecomment-621458997

(also, it was us who suggested the name `Corvus.Testing.AzureFunctions` 🙂)